### DISCUSSION TITLE
NextJS Build successful but no index file found

### DISCUSSION THREAD

**Discussion by 4nduril** [vercel/community#3824] [Help] [answered]
### Summary

Hi. I just tried to deploy a simple gallery website. I used the code several times before in other projects. The build runs successful and reports one static page ("/") to be generated. But when I visit the domain there is only a 404 with an infrastructure id (ID: fra1::9zkmv-1693329493061-9806c83a32c8). Also the deployment summary only lists the image files and says "No framework detected".
I turned on directory listings to see what's there and the folders with images and data are present. Nothing else. 

What could be the reason?

### Steps to Reproduce

The assigned domain is https://soa-23.vercel.app.

**Comment by amyegan** [comment]
Hey @4nduril. I don't have enough information here to know the exact cause. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give me a better view of what is going wrong.

**Comment by 4nduril** [comment, marked as answer]
Thank you so much @amyegan . I just removed the project completely from my vercel account and set it up again from the CLI and it worked. Maybe I messed some configuration in the dashboard up before. Thank you anyway for your responsse.

**Comment by MiguelTolosa** [comment]
Just ran into this same issue and fixed with deleting / recreating, may want to add this to the docs somewhere since it's a bug. Started with adding a new domain and creating a project from there instead of creating a project and then adding a new domain.
